packages/forms/docs/05-validation.md

```diff
@@ -481,6 +481,20 @@ TextInput::make('slug')->rules([
 ])
 ```
 
+You may also inject a $get callable to your closure rules, if you need to reference other fields on your form:
+
+```php
+TextInput::make('slug')->rules([
+    function (Get $get) {
+        return function (string $attribute, $value, Closure $fail) use ($get) {
+            if ($get('other_field') === 'foo' && $value !== 'bar') {
+                $fail("The {$attribute} is invalid.");
+            }
+        };
+    },
+])
+```
```

Comment:
```suggestion
You may inject utilities like `$get` into your custom rules, for example if you need to reference other field values in your form:

```php
use Closure;
use Filament\Forms\Get;

TextInput::make('slug')->rules([
    fn (Get $get): Closure => function (string $attribute, $value, Closure $fail) use ($get) {
        if ($get('other_field') === 'foo' && $value !== 'bar') {
            $fail("The {$attribute} is invalid.");
        }
    },
])
```
```